website/docs/history-and-hard-forks/history-vm-version.mdx

```diff
@@ -0,0 +1,212 @@
+---
+id: history-vm-version
+title: "History VM Version"
+---
+
+Usually, a hard fork will bring a new CKB-VM version in CKB network history for better security, performance, bug fixings and new RISC-V extensions. This article lists all the version of CKB-VM from previous hard fork history.
+
+One thing to be noted is that, even the hard fork brings upgrade to the network, it is important to prevent the hard fork from breaking the old Script codes. CKB makes best efforts to ensure users have the opt-in option to specify the VM version for their Script. For the specific mechanism that determines how the CKB node chooses the CKB-VM version for a transaction Script group, please refer to [vm-selection](/docs/script/vm-selection).
```

Comment:
```suggestion
The CKB network has introduced various CKB-VM versions over time to enhance security, performance, resolve bugs, and support new RISC-V extensions. This article lists all versions of the CKB-VM from previous hard fork events.

It is important to note that, even though a hard fork upgrades the network, it is crucial to prevent the hard fork from breaking the old Script codes. CKB makes best effort to ensure users have the opt-in option to specify the VM version for their Script. For the specific mechanism that determines how the CKB node chooses the CKB-VM version for a transaction's Script group, please refer to [vm-selection](/docs/script/vm-selection).
```